docs/device-agent/quickstart.md

```diff
@@ -7,109 +7,59 @@ navOrder: 2
 
 ## Add Device to FlowFuse
 
-### Install Device Agent Software
-
-1. Open a Terminal/Command Prompt as an root/administrative user
-2. Run the following commands in sequence:
-
-Windows:
-```bash
-mkdir c:\opt\flowfuse-device
-cd c:\opt\flowfuse-device
-npm install -g @flowfuse/device-agent
-flowfuse-device-agent --ui --ui-user admin --ui-pass password --ui-port 8081
-```
-Linux/MacOS:
-```bash
-mkdir /opt/flowfuse-device
-cd /opt/flowfuse-device
-npm install -g @flowfuse/device-agent
-flowfuse-device-agent --ui --ui-user admin --ui-pass password --ui-port 8081
-```
-
-Note: The flags used in the command above have the following meanings:
-- `--ui`: Start the device agents web-based user interface.
-- `--ui-user`: Specify the username for the web user interface.
-- `--ui-pass`: Specify the password for the web user interface.
-- `--ui-port`: Specify the port for the web user interface.
-- You can find more details about these flags [here](https://flowfuse.com/docs/device-agent/running/).
-
-### Add Device in FlowFuse
-1. Open your web browser and go to your FlowFuse application page.
-2. Navigate to the **Devices** section.
-3. Click **Add Device.**
-4. Fill in the **Device Name** and **Device Type,** then click **Add.**
+### Prerequisites
 
-<img src="images/add_device.png" width=500 />
+If you do not already have the device agent installed, follow the [install instructions](./install.md), otherwise, proceed to the next step.
 
-### Configure Device Agent
-1. Expand the Manual setup section, click the **Copy** button below the **Device Configuration**.
+_NOTE: It is good practice to keep your Device Agent up to date. Update instructions are also provided in the link above._
 
-<img src="images/config_yml2.png" width=500 />
+_NOTE: If you do not have access to a terminal, follow the [Quick Start with Web UI](./quickstart-web-ui.md) guide._
```

Comment:
_"If you do not have access to a terminal, follow the [Quick Start with Web UI]"_ where the first step is a command line entry?

Reply:
Yep, I realise that but it is difficult to describe without being overly verbose. 

The intention is that the Web UI can be used with terminal acces OR without (i.e. pre-installed)

Reply:
Understood, maybe advising that the Web UI allows for more manual control/customisation of the configuration instead, and directing accordingly?

Reply:
i'll have a go.

Reply:
Ah, i wonder if you are still seeing the old doc?

I set this as follows (specifically item 2 in the web-ui doc)

> 
> 1. If you have access to a Terminal/Command Prompt and run the following command:
>     ```bash
>     flowfuse-device-agent --ui --ui-user admin --ui-pass password --ui-port 8081
>     ```
>     _You can find more details about the Device Agent flags [here](https://flowfuse.com/docs/device-agent/running/)_
> 2. If the Device Agent is pre-installed on your hardware, follow the manufacturers instructions to start the device agent with the Web UI enabled.

Reply:
I was referring to the `NOTE: If you do not have access to a terminal, follow the [Quick Start with Web UI](http://localhost:8080/docs/device-agent/quickstart-web-ui) guide.` in the standard `quickstart.md`. Explanation with the `Web UI` doc is fine.

Reply:
Then i am not sure how else to word this.

I needed a way to link to the alternative setup (the web-ui setup) and this felt like the right place.

I deliberately did not add an entry in the index - i wanted folk to land in this doc (so they can see there is a quicker/default setup method) before venturing off to the web-ui version.

Perhaps we could change the note to:

> NOTE: If you prefer, follow the [Quick Start with Web UI](http://localhost:8080/docs/device-agent/quickstart-web-ui) guide

However, they may decide that is not relevant to them and thus get stuck looking for instructions relevant to a pre-installed env?


Or a bit more verbose:

> NOTE: If you prefer to use the Device Agent WEB UI, follow the [Quick Start with Web UI](http://localhost:8080/docs/device-agent/quickstart-web-ui) guide


Or a bit more explicit:

> NOTE: If your device agent is pre-installed and you do not have command line access, or you prefer to use the Device Agent WEB UI, then you can follow the [Quick Start with Web UI](http://localhost:8080/docs/device-agent/quickstart-web-ui) guide


Please advise.

Reply:
Option 2

Reply:
```suggestion
_NOTE: If you prefer to use the Device Agent WEB UI, follow the [Quick Start with Web UI](./quickstart-web-ui.md) guide._
```

@joepavitt to be sure we both agree, I will provide this update as a suggestion. Please apply if correct.